packages/editor/src/client/utility.ts

```diff
@@ -207,6 +207,14 @@ export function validateURL(url: string) {
   return false
 }
 
+export function flattenArray(array: unknown[] | DOMTokenList) {
```

Comment:
Not sure why we need this? This works exactly like `array.join(" ")`, let's use that in tsx.

Reply:
Since its a DOMTokenList there is no join("") function.....

Reply:
OK!